pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/release-notes/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes  
+excerpt: Discover the new features, updates, and known limitations in this release of the Hive Metastore Service
+title: Release Notes 
+menuWeight: 10
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+# Release Notes for the {{ model.techName }} Service version 1.0.0-3.0.0
+
+This is the first release of the {{ model.techName }}! Please read the documentation for details, but some highlights include:
+
+- Support for the following databases: Derby, MySQL, Percona-MySQL
+- Ability to easily configure each node's resources
+- Support for high-availability with the ability to horizontally scale out
+- Automatic generation of the `metastore-site.xml` file and exposure of it through the `endpoints` CLI command
+- Beeline CLI support
+- Use of the SDK version `0.57.0`, which enables all the DCOS 2.0 features such as:
+  - node draining
+  - quota enforcement
+- Support for the [DC/OS Storage Service](https://docs.d2iq.com/mesosphere/dcos/services/storage/)
```

Comment:
We have to remove this bit from the release notes. It turns out we're only on SDK 0.56.1. (Let's just say that instead). 

We'll have to do a future release on SDK 57

Reply:
Replied [here](https://github.com/mesosphere/dcos-docs-site/pull/2582/files#r342437848)
(Don't know why quote replying this message went into that thread)